backend/models/scene.js

```diff
@@ -16,53 +16,56 @@
  */
 
 "use strict";
-const mongoose = require("mongoose");
-const ModelFactory = require("./factory/modelFactory");
-const Schema = mongoose.Schema;
 const utils = require("../utils");
 const db = require("../handler/db");
 const ExternalServices = require("../handler/externalServices");
 const matrix = require("./helper/matrix");
 
-const schema = Schema({
-	_id: Object,
-	parents: []
-});
+function cleanOne(sceneToClean) {
```

Comment:
same here


Scene to clean is also confusing, you're cleaning the nodes here.